can-value.md

```diff
@@ -45,18 +44,16 @@ const outer = new SimpleMap({
 const keyObservable = canValue.bind(outer, "inner.key");
 ```
 
-Now if we use [can-reflect.getValue can-reflect.getValue()] and pass it
-`keyObservable`, we get `outer.inner.key`:
+Now if we read `keyObservable.value`, we get the value at `outer.inner.key`:
 
 ```js
-canReflect.getValue(keyObservable);
-// Returns "hello"
+keyObservable.value; // is "hello"
 ```
 
-We can also set the value with [can-reflect.setValue can-reflect.setValue()]:
+We can also set `keyObservable.value` to change the value at `outer.inner.key`:
 
 ```js
-canReflect.setValue(keyObservable, "aloha");
+keyObservable.value = "aloha";
```

Comment:
Here’s what this section looks like:
<img width="887" alt="screen shot 2018-07-02 at 7 28 22 pm" src="https://user-images.githubusercontent.com/10070176/42195598-7bb8e838-7e2e-11e8-87ab-20131b9fc252.png">